frontend/src/utils/__tests__/items.spec.ts

```diff
@@ -46,23 +46,23 @@ describe('getShapeFromCollectionType', () => {
 describe('getShapeFromItemType', () => {
   it('returns the correct card shape based on item type', () => {
     expect(getShapeFromItemType('Audio')).toEqual(CardShapes.Square);
-    expect(getShapeFromItemType('folder')).toEqual(CardShapes.Square);
-    expect(getShapeFromItemType('musicalbum')).toEqual(CardShapes.Square);
-    expect(getShapeFromItemType('musicartist')).toEqual(CardShapes.Square);
-    expect(getShapeFromItemType('musicgenre')).toEqual(CardShapes.Square);
-    expect(getShapeFromItemType('photoalbum')).toEqual(CardShapes.Square);
-    expect(getShapeFromItemType('playlist')).toEqual(CardShapes.Square);
-    expect(getShapeFromItemType('video')).toEqual(CardShapes.Square);
+    expect(getShapeFromItemType('Folder')).toEqual(CardShapes.Square);
```

Comment:
I don't care about getting this change in or out, but I was actually thinking on removing all the test files before merging and do proper testing from the start using Vitest or Cypress (which was the next major thing I wanted to tackle after Vue 3).

If you are going to be around and want to help in the testing suite, I'll leave this up to you (maybe you prefer to incrementally refactor, instead of a complete redo?) :).

Reply:
I'd prefer to keep it in, I'm a fan of incremental changes in the right direction. I think having these old test cases around will help when the test suite is upgraded. I hope I'll still be around to help with that!

Reply:
@aweebs Okay. Be aware anyway that removing them will still make them available in history, so we're not losing them really. It's just a matter of deciding between how easy it will be to review new files vs or incrementally modify them (because if the modifications are going to be huge, maybe it's not worth it).

I looked at vitest API and it looks really similar to jest, so I'm not sure what to think (this is why I'm asking :P)